client/stylesheets/base/global.less

```diff
@@ -16,23 +16,24 @@
 @import '@{components}/buttons.import.less';
 
 
-a.tag-label span:hover {
-    background-color: #E97F3C;
-    text-decoration: none;
+body {
+    background-color: #f2f2f2;
 }
 
-a.tag-label:hover {
-    text-decoration: none;
+.course-whole-page {
```

Comment:
Also, the 'whole-page' part of the class name is [considered non-semantic](https://css-tricks.com/semantic-class-names/) and does not pertain to the actual style rules anyway. 

Is there a way to simply use Bootstrap panel or container to achieve the whole page outcome?